src/index.js

```diff
@@ -0,0 +1,40 @@
+// eslint-disable-next-line no-unused-vars
+import _ from 'lodash';
+import './style.css';
+import compare from './sorted';
+
+const todoLists = [
+  {
+    description: 'Working on javascript todo app',
+    completed: false,
+    index: 1,
+  },
+  {
+    description: 'Work on ruby on rails',
+    completed: true,
+    index: 2,
+  },
+  {
+    description: 'Work on php laravel',
+    completed: true,
+    index: 8,
+  },
+  {
+    description: 'Work on java',
+    completed: false,
+    index: 6,
+  },
+];
+
+const sorted = todoLists.sort(compare);
+const ul = document.getElementById('todo-lists');
+
+window.onload = () => {
+  sorted.forEach(todo => {
+    const li = document.createElement('li');
+    li.setAttribute('class', 'one-task');
+    li.setAttribute('draggable', 'true');
+    li.innerText = todo.description;
+    ul.appendChild(li);
+  });
```

Comment:
- [ ] [OPTIONAL] I think it could be a good idea to extract the logic of rendering the list of tasks into a separate function that you can call inside the `window.onload`, this is because in the future you may have other logic you will need to add in the `window.onload` so to keep the code clean it could be good to only call functions inside of it instead 👍